### ISSUE TITLE
License detection crashes on LICENSE file

### ISSUE THREAD

**Issue by dfellis** [benbalter/licensee#330]
### Describe the bug

When I manually run `licensee` on [this repo](https://github.com/dfellis/multitransport-jsonrpc), the application crashes in detect.rb line 96. It appears to be affected by the `LICENSE` file; if I manually detect only the `package.json` license assertion it succeeds.

### Steps to reproduce the behavior

1. Clone [this repo](https://github.com/dfellis/multitransport-jsonrpc)
2. Run `licensee`, `licensee detect LICENSE`, and `licensee detect package.json` in the root of the repo.
3. The first two fail similarly, the third suceeds.

### Expected behavior

The MIT license should be detected in all scenarios.

### "Screenshots"

```bash
damocles@Talyn:~/oss/multitransport-jsonrpc(master)$ licensee
License:        NOASSERTION
Matched files:  LICENSE, package.json
LICENSE:
  Content hash:  b4df03dcc4b1ad2e45be05deb6206694bf5241ee
  License:       NOASSERTION
Traceback (most recent call last):
	10: from /usr/local/bin/licensee:23:in `<main>'
	 9: from /usr/local/bin/licensee:23:in `load'
	 8: from /var/lib/gems/2.5.0/gems/licensee-9.9.4/bin/licensee:37:in `<top (required)>'
	 7: from /usr/lib/ruby/vendor_ruby/thor/base.rb:444:in `start'
	 6: from /usr/lib/ruby/vendor_ruby/thor.rb:369:in `dispatch'
	 5: from /usr/lib/ruby/vendor_ruby/thor/invocation.rb:126:in `invoke_command'
	 4: from /usr/lib/ruby/vendor_ruby/thor/command.rb:27:in `run'
	 3: from /var/lib/gems/2.5.0/gems/licensee-9.9.4/lib/licensee/commands/detect.rb:37:in `detect'
	 2: from /var/lib/gems/2.5.0/gems/licensee-9.9.4/lib/licensee/commands/detect.rb:37:in `each'
	 1: from /var/lib/gems/2.5.0/gems/licensee-9.9.4/lib/licensee/commands/detect.rb:52:in `block in detect'
/var/lib/gems/2.5.0/gems/licensee-9.9.4/lib/licensee/commands/detect.rb:96:in `licenses_by_similiarity': undefined method `licenses_by_similiarity' for #<Licensee::Matchers::Dice:0x0000564605d0f768> (NoMethodError)
Did you mean?  licenses_by_similarity
damocles@Talyn:~/oss/multitransport-jsonrpc(master)$ licensee detect LICENSE
License:        NOASSERTION
Matched files:  LICENSE
LICENSE:
  Content hash:  b4df03dcc4b1ad2e45be05deb6206694bf5241ee
  License:       NOASSERTION
Traceback (most recent call last):
	10: from /usr/local/bin/licensee:23:in `<main>'
	 9: from /usr/local/bin/licensee:23:in `load'
	 8: from /var/lib/gems/2.5.0/gems/licensee-9.9.4/bin/licensee:37:in `<top (required)>'
	 7: from /usr/lib/ruby/vendor_ruby/thor/base.rb:444:in `start'
	 6: from /usr/lib/ruby/vendor_ruby/thor.rb:369:in `dispatch'
	 5: from /usr/lib/ruby/vendor_ruby/thor/invocation.rb:126:in `invoke_command'
	 4: from /usr/lib/ruby/vendor_ruby/thor/command.rb:27:in `run'
	 3: from /var/lib/gems/2.5.0/gems/licensee-9.9.4/lib/licensee/commands/detect.rb:37:in `detect'
	 2: from /var/lib/gems/2.5.0/gems/licensee-9.9.4/lib/licensee/commands/detect.rb:37:in `each'
	 1: from /var/lib/gems/2.5.0/gems/licensee-9.9.4/lib/licensee/commands/detect.rb:52:in `block in detect'
/var/lib/gems/2.5.0/gems/licensee-9.9.4/lib/licensee/commands/detect.rb:96:in `licenses_by_similiarity': undefined method `licenses_by_similiarity' for #<Licensee::Matchers::Dice:0x000055d82bfb1608> (NoMethodError)
Did you mean?  licenses_by_similarity
damocles@Talyn:~/oss/multitransport-jsonrpc(master)$ licensee detect package.json
License:        MIT
Matched files:  package.json
package.json:
  Confidence:  90.00%
  Matcher:     Licensee::Matchers::NpmBower
  License:     MIT
damocles@Talyn:~/oss/multitransport-jsonrpc(master)$
```

**Comment by benbalter** [comment]
Thanks for the great report @dfellis! It was a typo. Fixed via #331.

**Comment by dfellis** [comment]
:+1:
